tds-item-selection-components/src/main/java/tds/itemselection/selectors/impl/AdaptiveSelector2013.java

```diff
@@ -41,11 +41,7 @@
   private static Logger logger = LoggerFactory.getLogger(AdaptiveSelector2013.class);
 
   // a combination of test-constant and examinee-variable data
-  private Blueprint blueprint;
-  private Cset1 cset1;
-  private ItemCandidatesData itemCandidates;
-  private TestSegment segment;
-  private Random rand = new Random();
```

Comment:
[Error](http://logstash-loadtest.sbtds.org/_plugin/kibana/app/kibana#/doc/logstash-*/logstash-2017.06.05/logs?id=AVx501NmxZF4Xlf0EA00&_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-15m,mode:quick,to:now))).